packages/react-slider/src/utils/calculateSteps.ts

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { clamp } from './clamp';
+
+/**
+ * Calculates the `step` position based off of a `Mouse` or `Touch` event.
```

Comment:
Can you add a little more info about what this returns? Is it the position on the screen or something else?